ui/src/layouts/storage/files_layout/file_preview.rs

```diff
@@ -129,6 +136,15 @@ fn FilePreview<'a>(cx: Scope<'a, Props<'a>>) -> Element<'a> {
     let local_disk_path_fixed =
         get_fixed_path_to_load_local_file(file_path_in_local_disk.read().clone());
 
+    let code_content = if is_code {
+        match fs::read_to_string(file_path_in_local_disk.read().clone()) {
+            Ok(content) => content,
+            Err(_) => String::new(),
+        }
+    } else {
+        String::new()
+    };
+
```

Comment:
We can simplify this logic to something like the following

```suggestion
let code_content = is_code
        .then(|| std::fs::read_to_string(file_path_in_local_disk.read().clone()).ok())
        .flatten()
        .unwrap_or_default();

```